app/classifier/index.cjsx

```diff
@@ -655,6 +672,8 @@ module.exports = React.createClass
       <Classifier {...@props}
         workflow={@props.workflow}
         subject={@state.subject}
+        guide={@state.guide}
```

Comment:
Oh, I think the request in the classifier wrapper component in the index file is fine for now, though we may get feedback from volunteers that were using it on Talk after this is merged. I just mean that the guide and images are being passed down as props to the classifier component where those props aren't being used.